api/provider/aws/logging.go

```diff
@@ -0,0 +1,165 @@
+package aws
+
+import (
+	"strings"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/cloudwatchlogs"
+	"github.com/aws/aws-sdk-go/service/cloudwatchlogs/cloudwatchlogsiface"
+	"github.com/quintilesims/layer0/common/models"
+)
+
+const (
+	// DescribeLogStreams is throttled after five transactions per second.
+	// With 50 streams/transaction, 1000 gives a reasonable streams:time ratio
+	MAX_DESCRIBE_STREAMS_COUNT = 1000
+)
+
+func GetLogsFromTaskARNs(
```

Comment:
This seems like a difficult function to work with from the caller's perspective

Reply:
I think it's about as simple as one can make it given we have all of these tail/start/end params. 
If you look at the `taskProvider.Logs()` caller, it's actually quite simple IMO.

Reply:
taking a step back, do we even need all of the start/tail/end params?

Reply:
yeah, the `log` command becomes pretty unusable without at least one of them - think about doing `l0 service logs api` when your api has been up for a month

Reply:
I'm not suggesting we don't need at least one of these params, just questioning if we need all of them. "Complex" date range filtering tasks are better done within CloudWatch itself or, in the future, Splunk. On the cli, all I would want is a `start` param to specify how far back the logs should potentially go.

Reply:
well, it's already been implemented and it works, so i don't see why we should take it out.